src/executor/Executor.hxx

```diff
@@ -87,7 +87,7 @@ public:
      * the execution is completed. @param fn is the closure to run. */
     void sync_run(std::function<void()> fn);
 
-#ifdef __FreeRTOS__
+#if defined(__FreeRTOS__) && !defined(ESP32)
```

Comment:
We should revisit this.  Should we define __FreeRTOS__ as part of the Arduino build?  ...same comment for those that follow.

Reply:
yes